kube/albuquery-ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
+  name: albuquery-ingress
+spec:
+  rules:
+  - host: lsst-pdac.ncsa.illinois.edu
```

Comment:
This is probably fine for now, but if we deploy to another instance (e.g. `lsst-dac.in2p3.fr`) - how should we manage this parameter or `albuquery-ingress.yaml`

Reply:
That's an interesting point for a few reasons:

1) This was never mentioned as a requirement.  I didn't even know we'd want to deploy this somewhere other than NCSA.  Is there another cluster there that we are also running?

2) Is this something I should get an account on?

What we can do if this is a requirement / when it becomes a requirement:

1) Relax this to be all hosts, which I think is possible by using a *.  But if you use any hostname that isn't lsst-pdac.ncsa.illinois.edu, you'll have problems resolving the certificate on the https connection.  If there's another way people try to contact the service that isn't the approved SSL one, I really don't want to let them in, it will only cause problems.
2) Have another ingress file for the different environment, which may also be configured differently?  We could have the scripts deploy to either environment.

If there are different databases to connect to, with different passwords, then also the secrets files will need to be created there with the proper config.

Reply:
I talked to @fritzm and I think we just make the host a config map thing in the future when we need an additional deployment.

Additional deployments might be at NCSA, IN2P3, Google or whatever, but it's fine for now.